docker/lr-jupyter/setupBigJupyterHost.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
```

Comment:
I'm having a hard time understanding why this is needed

Reply:
Move to scripts dir.  Maybe remove this altogether…